cypress/integration/pages/home_page_spec.ts

```diff
@@ -0,0 +1,32 @@
+/// <reference types="cypress" />
+export {};
+
+describe('Test Home Page', () => {
+  beforeEach(() => {
+    cy.visit('/');
+  });
+  it('renders the hero title', () => {
+    cy.contains('Find, Share and Publish Quality Data with Datahub');
+  });
+  it('checks that a search form exists', () => {
+    cy.get('form').type('dataset').contains('Search');
+  });
+  it('submits the search form', () => {
+    cy.get('form').find('[type="text"]').type('my-dataset');
+    cy.get('form').submit();
+    cy.url().should('include', '/search?q=my-dataset&sort=');
+    cy.get('.text-3xl')
+      .should('be.visible')
+      .should('have.text', '366 results found');
```

Comment:
Since you're testing against live CKAN API, this number will change almost every day.

Reply:
@anuveyatsu This is true and I will check how to make it flexible.